shared/src/socket/socketio.ts

```diff
@@ -27,16 +41,17 @@ class Socketio {
     private socket: Socket;
     private retryTimeout: number = 0;
     private localChanges = db.getLocalChangesAsRef();
-    private isCms: boolean;
+    private isCMS = false;
```

Comment:
For consistency, variables should be all lowercase, except the first letter of each distinct word (camelCase). So `isCms` is the "correct" way of spelling "is CMS".